pkg/addons/default/kube_proxy.go

```diff
@@ -55,6 +57,12 @@ func UpdateKubeProxyImageTag(clientSet kubernetes.Interface, controlPlaneVersion
 		}
 		return false, errors.Wrapf(err, "getting %q", KubeProxy)
 	}
+
+	hasArm64NodeSelector := daemeonSetHasArm64NodeSelector(d)
+	if !hasArm64NodeSelector {
+		logger.Info("missing arm64 nodeSelector value")
```

Comment:
Does it make sense to log this down on 102 where we do the adding?

Reply:
we do a log first as we might exit earlier if `plan` is `false`

Reply:
ah i see